docs/ManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -218,11 +243,22 @@ You can uninstall Scalar DL with the following Helm commands:
 
    ```console
     # Uninstall loaded schema with a release name 'load-schema'
-      helm uninstall load-schema
+    helm uninstall load-schema
 
     # Uninstall Scalar DL with a release name 'my-release-scalardl'
-      helm uninstall my-release-scalardl
+    helm uninstall my-release-scalardl
    ```
+
+You can uninstall Scalar Auditor with the following Helm commands:
+
+    ```console
```

Comment:
This code block is not properly formatted. 4-space indentation is not needed here.

In addition, [the another code block above](https://github.com/scalar-labs/scalar-kubernetes/pull/150/files#diff-77985a58e1a44d3a5f553f0ca5a7a99987a85eb373b09eb41c8d81de01acf8bbR244) is also not needed to be indented for the same reason. (It's formatted well because it's indented with 3 spaces.)

Reply:
Sorry for the inconvenience.
Above specified changes are integrated e7bc2c2